### ISSUE TITLE
WP1 Wikiproject selection is really not straight

### ISSUE THREAD

**Issue by kelson42** [openzim/wp1#775]
At https://wp1.openzim.org/#/selections/wikiproject, there is no helper and only a link to [a page](https://en.wikipedia.org/wiki/WikiProject) which does not even list all the projects.

We need an easier way to get the strings which could be included in the two texteareas.

**Comment by audiodude** [comment]
Yes we already have the WikiProject autocomplete selector that we use on the homepage:

![Image](https://github.com/user-attachments/assets/83f6b541-4eba-4376-84dc-25512f6731b1)

We should definitely just use that, with some kind of "Add" button that adds the project to the list as a chip, along with "[x]" buttons on the chips to remove them from either list.

I think this was coded in like two hours at a hackathon so that's probably why it doesn't exist. 😉

**Comment by nicolapace** [comment]
Published something here #841
